src/services/chat/prepopulated.js

```diff
@@ -41,17 +41,17 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser
+                  ...ownUser.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: ownUser?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
               },
               {
                 user: {
-                  ...user
+                  ...user.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: user?.user_id,
                 is_anonymous: false,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `is_anonymous` field is hardcoded to `false`. If this is intentional and all users in this context are expected to be non-anonymous, then this is fine. Otherwise, you might want to consider making this dynamic based on the user's actual anonymity status.



<!-- This is an auto-generated comment by OSS CodeRabbit -->